extras/systemd/dynflow-sidekiq@.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Foreman jobs daemon on sidekiq
```

Comment:
This is the executor process? Does it matter that its sidekiq based?

Reply:
This is eighter orchestrator or worker process, depending only on given configuration for such process. (`-C %i.yml` part in `ExecStart`)

Reply:
Could you expand a bit further on how that plays out? If I look at my system, am I able to look and know this process is my executor and these processes are my workers?

Reply:
Yeah,
you should have two services by default:
1. `dynflow-sidekiq@orchestrator` service - having config in `/etc/foreman/dynflow/orchestrator.yml`
1. `dynflow-sidekiq@worker` service - having config in `/etc/foreman/dynflow/worker.yml`

Those are the only two running by default, but you can make new config file `/etc/foreman/dynflow/worker2.yml` and run `dynflow-sidekiq@worker2` service.